generate.sh

```diff
@@ -2,13 +2,14 @@
 
 virtualenv --python=python yaydocvenv
 source yaydocvenv/bin/activate
-pip install sphinx
+pip install sphinx==1.5.5
```

Comment:
Instead of these pip statements, can we please use a `requirements.txt` file to track dependencies ?